torch/onnx/_internal/exporter/_building.py

```diff
@@ -458,11 +458,25 @@ def _process_python_sequences(
     return named_inputs
 
 
+def _determine_output_number(
+    signature: _schemas.OpSignature, named_attrs: Mapping[str, ValidAttributeType]
+) -> int:
+    """Determine the number of outputs for the node with heuristics."""
+    if signature.domain == "":
+        if signature.name == "BatchNormalization":
+            if not named_attrs.get("training_mode", 0):
+                return 1
+        if signature.name == "Split":
+            return named_attrs.get("num_outputs", 2)  # type: ignore[return-value]
```

Comment:
Is there any other way to do this?

Reply:
An alternative I thought of was to define a special arg eg _num_outputs for op calls that will direct the builder to create this number of outputs. This will match our experience in the rewriter.

Reply:
Removed default